dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v41/V2_41_46__add_data_sharing_for_aggregate_data_exchange.java

```diff
@@ -40,8 +40,8 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * In table aggregatedataexchange, copy Metadata Write value to Data Write value in sharing.users and
- * sharing.userGroups in sharing column. Public access must not be changed.
+ * In table aggregatedataexchange, copy Metadata Write value to Data Write value in sharing.users
+ * and sharing.userGroups in sharing column. Public access must not be changed.
```

Comment:
Update flyway script will cause checksum error. Unless its really necessary, I think we should remove this change.

Reply:
yeah, good catch. I can revert, but the formatting step will fail. Let's see if it allows merging with an invalid format.
I wonder how the unaccepted format got in?

Reply:
ah, [the original PR](https://github.com/dhis2/dhis2-core/pull/16558) did have a format error but it was merged anyway

Reply:
ah, I thought the formatter error would block the Merge button ? I will create another PR to fix it.

Reply:
maybe we should make the formatting a mandatory PR step, needing to pass. We can chat about that next week as a team and as a wider group I suppose.